spec/models/news/item_spec.rb

```diff
@@ -60,6 +60,14 @@
       end
 
       it { is_expected.to eq collections.map(&:id).reverse }
+
+      context 'with priorities' do
+        before { item.add_collection priority }
+
+        let(:priority) { create(:news_collection, name: 'CCC', priority: 1) }
+
+        it { is_expected.to eq [priority.id] + collections.map(&:id).reverse }
```

Comment:
This test is a bit cryptic,
what are the other priority ids?
should we check also the name attr here? (in case of duplicated id?)

Reply:
Moved to a `let` block to assign a name